README.md

```diff
@@ -1,2 +1,11 @@
 # proto
+
 Shared Protocol Buffers and their associated generated code
+
+## Working on this repo
+
+Before committing any changes to this repo, you will want to run `npm i` at least once. This will install the Husky precommit hooks to your machine.
+
+Go code will be regenerated based on the `.proto` files on each commit and does not need to be done manually.
+
+Commit messages must be in the [Angular Commit Message Convention](https://gist.github.com/stephenparish/9941e89d80e2bc58a153)
```

Comment:
Out of curiosity: Is this used to generate something? Or just a preference for the repo?

Reply:
It gets used to version the NPM package, just like XMTP JS

Reply:
Nice! Might be useful to add that note to the README for clarity on how it's used, but it could be declared somewhere else and I'm not seeing it.